subject.py

```diff
@@ -0,0 +1,178 @@
+import datajoint as dj
+
+import reference
+
+schema = dj.schema(dj.config['names.%s' % __name__], locals())
+
+
+
+@schema
+class Species(dj.Manual):
+    # <class 'subjects.models.Species'>
+    definition = """
+    species_id:			int             # species id
```

Comment:
given binomial info will be inserted, using arbitrary `species_id` will only make downstream tables harder to use. I suggest replacing `species_id` with `binomial`

Reply:
makes sense